cookbooks/convection-box.prm

```diff
@@ -101,7 +101,9 @@ end
 
 subsection Formulation
   set Formulation = custom
-  set Temperature density approximation = adiabatic
+  set Mass conservation    = incompressible
+  set Temperature equation = real density #reference density profile
```

Comment:
oh, to remind me that I need to think about what formulation we want. Do we want to keep it as our weird "not-quite Boussinesq"?